app/frontend/components/shared/permit-applications/permit-application-status-tabs.tsx

```diff
@@ -14,41 +14,48 @@ export const PermitApplicationStatusTabs = observer(function ToggleArchivedButto
 }: IPermitApplicationStatusTabsProps<TSearchModel>) {
   const { permitApplicationStore } = useMst()
   const queryParams = new URLSearchParams(location.search)
-  const paramStatusFilter = queryParams.get("status") as TFilterableStatus
+  const paramStatusFilterString = queryParams.get("status") as string
+  const paramStatusFilter = paramStatusFilterString?.split(",") as TFilterableStatus[]
 
-  const statusToIndex = (status: EPermitApplicationStatus): number => {
-    const index = Object.values(EPermitApplicationStatus).indexOf(status)
-    return index === -1 ? 0 : index
+  const draftFilters = [EPermitApplicationStatus.newDraft, EPermitApplicationStatus.revisionsRequested]
+  const submittedFilters = [EPermitApplicationStatus.newlySubmitted, EPermitApplicationStatus.resubmitted]
+
+  const statusStringToIndex = (statusString: string): number => {
+    const map = {
+      [draftFilters.join(",")]: 0,
+      [submittedFilters.join(",")]: 1,
+    }
+    return map[statusString]
```

Comment:
what if the status string order is not as expected. Since it's a queryParam, anyone can edit those in the url bar